var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -28,13 +28,6 @@ class Crtm(CMakePackage):
     variant(
         "fix", default=False, description='Download CRTM coeffecient or "fix" files (several GBs).'
     )
-    variant(
-        "build_type",
-        default="RelWithDebInfo",
```

Comment:
`crtm` has a different default

Reply:
@climbfuji do you remember what the RelWithDebInfo settings was for? Is it okay if we remove this block, then we can set the default in spack-stack if needed?

Reply:
There's https://github.com/JCSDA/spack-stack/issues/827 which requests changing crtm to `Release`, so I think we should go ahead and make this change. Thanks!

Reply:
I agree that it should build to `Release` by default for spack stack